python/tests/test_routers.py

```diff
@@ -95,128 +95,8 @@ def tearDown(self):
         Debug().events_received.clear()
         Debug().events_published.clear()
 
-    def test_head_on_root_without_redirect(self):
```

Comment:
Add similar v1 tests.

Reply:
Then please write an issue about v1 endpoint code not being covered.

Reply:
Out of scope for this PR. If you wish to have an issue for writing these tests then I think you are better suited to describe what you want :)